Functions/xASL_adm_RemoveDirectories.m

```diff
@@ -0,0 +1,36 @@
+function xASL_adm_RemoveDirectories(root)
+%xASL_adm_RemoveDirectories Script to remove all ExploreASL related paths
+%
+% FORMAT:       xASL_adm_RemoveDirectories(root)
+% 
+% INPUT:        root - Root directory (REQUIRED, CHAR ARRAY)
+%
+% OUTPUT:       n/a
+% 
+% -----------------------------------------------------------------------------------------------------------------------------------------------------
+% DESCRIPTION:  Script to remove all ExploreASL related paths.
+%
+% -----------------------------------------------------------------------------------------------------------------------------------------------------
+% EXAMPLE:      xASL_adm_RemoveDirectories(x.MyPath);
+% __________________________________
+% Copyright 2015-2021 ExploreASL
+
+    %% Check input
+    if nargin < 1
+        error('Please provide a root directory...');
+    end
+    
+    % ExploreASL
+    try
+        warning('off', 'all');
```

Comment:
This is tricky, as it disables all warnings (so debugging becomes tricky) and enables all warnings (even those that a user might have disabled!

So this is not good practice. You can disable warnings specifically, google it or check in our current code. You run this script, then when a warning occurs, run `lastwarn` which gives you also the warning ID, and then you can disable this specific warning ID.

Reply:
8cdb69b